products/health-care/beneficiary-travel/engineering/spikes-for-api-team/eligible-appointments.md

```diff
@@ -0,0 +1,51 @@
+
+# {WIP} Problem: API endpoint to get `eligible appointments`
+
+- Currently, we are foreseeing a UX problems were we could be giving Veterans an option to submit a claim with a low chance of getting approved.
+- How can we drive the creation of claims that are more likely to be successful
+
+## Risks
+
+- Currently we stand the risk of allowing creation of claims that we predetermine if they will be approved or not
+  - Things like
+    - same day claims
+    - facility too close claims
+    - appointment not eligible claims
+- If we let Veterans create low-chance/invalid claims, we will
+  - loose Veteran confidence
+  - Create extra work for staff members
+  - overwhelm the system with junk claims
+
+## Proposed Spike
+
+One idea that is one the table is how can give clients some guidance into what appointments have a higher change of getting approved by creating an endpoint that allows for querying for what appointments are eligible
```

Comment:
Another idea I had boils down to: How can we expose or leverage the rules engine through the API to maintain policy/business adherence, and also give Veterans the most value.

Reply:
This idea might be a different-yet-related spike. @mdewey let me know if you want me to write that up.

Reply:
I am not sure if its a different ticket. I see you saw the question below. I think it should considered part of this ticket because ideally, we are not duplicating logic and re-using whats already been created and as `option b` create `a rules engine lite` that could be run here.